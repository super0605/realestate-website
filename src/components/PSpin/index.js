import React from "react"

const PSpin = () => {
  return (
    <Spin tip="Loading...">
      <Alert
        message="Alert message title"
        description="Further details about the context of this alert."
        type="info"
      />
    </Spin>
  )
}

export default PSpin;