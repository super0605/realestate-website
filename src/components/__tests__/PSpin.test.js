import * as React from "react";
import { mount, shallow } from "enzyme";
import PSpin from "../PSpin";

describe("PSpin Component", () => {
  let props;
  let mountedPSpin;
  let wrapper;

  const renderWrapper = (bShallow) => {
    if (!mountedPSpin) {
        mountedPSpin = bShallow
        ? shallow(<PSpin {...props} />)
        : mount(<PSpin {...props} />);
    }
    return mountedPSpin;
  };

  beforeEach(() => {
    wrapper = renderWrapper(true);
  });

  it("should render without problems", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
