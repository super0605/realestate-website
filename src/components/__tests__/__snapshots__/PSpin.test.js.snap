// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PSpin Component should render without problems 1`] = `
<Spin
  size="default"
  spinning={true}
  tip="Loading..."
  wrapperClassName=""
>
  <Alert
    description="Further details about the context of this alert."
    message="Alert message title"
    type="info"
  />
</Spin>
`;
